.github/filters.yaml

```
@@ -0,0 +1,13 @@
+backend:
+  - 'packages/**/package.json'
+  - 'packages/**/server/**/*.(js|ts)'
+  - 'packages/**/strapi-server.js'
+  - 'packages/{utils,generators,cli,providers}/**'
+  - 'packages/core/*/{lib,bin,ee}/**'
+  - 'api-tests/**'
+frontend:
+  - 'packages/**/package.json'
+  - 'packages/**/admin/src/**'
```

I think this is missing ee:

```suggestion
  - 'packages/**/admin/src/**'
  - 'packages/**/admin/ee/admin/**'
```

---

Wont `packages/**/admin` include `packages/core/admin/ee/admin` ? 🤔